morse_module/morse.c

```diff
@@ -101,6 +117,17 @@ static void do_blink(){
 	}
 }
 
+static void init_i2c_slave() {
```

Comment:
please don't have both `i2c_slave_init` and `init_i2c_slave` in the same project

Reply:
changed